frontend/src/components/common/ConditionalButton.js

```diff
@@ -16,19 +16,30 @@ const propTypes = {
   type: PropTypes.string,
   isDropdown: PropTypes.bool,
   overlay: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+  ghost: PropTypes.bool,
+  popConfirm: shape({
+    placement: PropTypes.string,
+    title: PropTypes.string,
+    onConfirm: PropTypes.func.isRequired,
+    okText: PropTypes.string,
+    cancelText: PropTypes.string
+  })
 };
 
 const defaultProps = {
+  handleAction: null,
```

Comment:
`null` is not a function

Default should be of the same type, so you can just use:
```javascript
import { noop } from "lodash";

const defaultProps = {
  handleAction: noop, // or handleAction: () => {},
};
```